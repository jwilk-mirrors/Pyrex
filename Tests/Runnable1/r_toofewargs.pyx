cdef class Spam:

	def __init__(self, a, b, int c):
		pass
