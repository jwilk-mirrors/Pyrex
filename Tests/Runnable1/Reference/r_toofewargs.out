TypeError: function takes exactly 3 arguments (0 given)
